docs/3.0rc/resources/upgrade-prefect-3.mdx

```diff
@@ -333,4 +333,37 @@ print(my_flow())  # Output: Failed(message='Flow failed due to task failure')
 ```
 </CodeGroup>
 
-Choose the strategy that best fits your specific use case and error handling requirements.
\ No newline at end of file
+Choose the strategy that best fits your specific use case and error handling requirements.
+
+## Common Errors
+
+#### "can't be used in 'await' expression"
+
+In Prefect 3, certain methods that were asynchronous in Prefect 2, such as `submit`, `map` (on `Task`), and `result`, `wait` (on `PrefectFuture`), are now synchronous. Attempting to use `await` with these methods will result in a `TypeError`, like:
```

Comment:
Might be worthwhile to use bullets to list the affected methods for readability.